lib/maths/CPeriodicityHypothesisTests.cc

```diff
@@ -281,7 +286,7 @@ void project(const TFloatMeanAccumulatorVec& values,
              core_t::TTime bucketLength,
              TFloatMeanAccumulatorVec& result) {
     result.clear();
-    if (!values.empty()) {
+    if (values.size() > 0) {
```

Comment:
Our standards now say to not use `!` but instead `== false` to match Elastic Java style. I personally find size() > 0 easier to read than empty() == false at a glance. I changed to be more compliant with our style guidelines because I was making quite a few changes in this file.